backend/.scripts/analyze_package_size.sh

```
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Chalice 배포 패키지 크기 분석 스크립트
+# backend 디렉토리로 이동 (스크립트는 backend/.scripts/에 위치)
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)